bin/run_tests

```diff
@@ -341,11 +338,6 @@ test_rust_sdk() {
 }
 
 test_integration() {
-    run_docker_test test_xo_smoke_all
```

Comment:
Is there anywhere a test like this can run? Its goal is to verify that all the xo transaction processors do the same thing (in particular, that they serialize everything the same way), which no other test does.

Reply:
This effort is non-trivial since some of the SDKs are moving out of the core repository into their own repos. It's on our list of to-dos.